example_config_usage.py

```
@@ -0,0 +1,161 @@
+    print("=" * 60)
+    print("🔑 环境变量替换")
+    print("=" * 60)
+    
+    config = get_config_manager()
+    
+    # 获取包含环境变量的配置
+    llm_config = config.get_module_config("llm")
+    api_key = llm_config["clients"]["openai"]["api_key"]
+    
+    print(f"🔑 API Key: {api_key}")
+    print(f"📝 如果设置了 OPENAI_API_KEY 环境变量，会被替换")
```

Ruff (0.15.9)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)